src/main/java/org/example/MyPortletUI.java

```diff
@@ -22,11 +22,10 @@
         service = UI.class,
         property = {
         "com.liferay.portlet.display-category=category.sample",  // Widget category
-        "javax.portlet.name=MyVaadinPortlet",                    // Unique portlet name in Liferay
+        "javax.portlet.name=example-portlet-1.1.0",       // Unique portlet name in Liferay
```

Comment:
Nit: I'd keep the portlet name consistent with the <artifactId> set in the pom for consistency
```suggestion
        "javax.portlet.name=example-osgi-portlet-1.1.0",       // Unique portlet name in Liferay
```